builder/proxmox/clone/config.go

```diff
@@ -11,8 +17,19 @@ import (
 type Config struct {
 	proxmox.Config `mapstructure:",squash"`
 
-	CloneVM   string         `mapstructure:"clone_vm"`
+	CloneVM   string         `mapstructure:"clone_vm" required:"true"`
 	FullClone config.Trilean `mapstructure:"full_clone" required:"false"`
+
+	Nameserver   string              `mapstructure:"nameserver" required:"false"`
+	Searchdomain string              `mapstructure:"searchdomain" required:"false"`
+	Ipconfigs    []cloudInitIpconfig `mapstructure:"ipconfig" required:"false"`
+}
+
+type cloudInitIpconfig struct {
```

Comment:
On a side-note, if this is linked to the `NICConfig`, is there a reason why this is specific to the `clone` builder? I wonder if that could apply to the `ISO` builder as well.

Reply:
At least right now, it's only supported for `clone` API-wise: https://github.com/Telmate/proxmox-api-go/blob/master/proxmox/config_qemu.go#L92